sale_order_import/wizard/sale_order_import.py

```diff
@@ -467,8 +467,16 @@ def _prepare_create_order_line(
                 "company_id": company_id,
             }
         )
+
         if price_source == "order":
-            vals["price_unit"] = import_line["price_unit"]  # TODO : fix
+            if "price_unit" not in import_line.items():
+                raise UserError(
+                    _(
+                        "No price were defined in the file. Please select a "
+                        "valid price before importing!"
```

Comment:
```suggestion
```

Reply:
Hi @nilshamerlinck, this PR is ready for review: Thank you!